INFO: 2020-02-15 09:22:08,246: perform_upgrade() 'Last run version was 0.6.7 or earlier, current version is 0.6.7'
INFO: 2020-02-15 09:22:08,247: process_command() 'Received command: []'
INFO: 2020-02-15 09:22:08,254: load_banned() 'Missing or invalid banned URLs list, no URLs will be banned'
INFO: 2020-02-15 09:22:08,778: load() 'Jumble loading'
INFO: 2020-02-15 09:22:08,779: _walk_modules() 'Jumble loading module in __init__ from /usr/share/variety/plugins/__init__.py'
INFO: 2020-02-15 09:22:08,807: _walk_modules() 'Jumble loading module in LocalFilesSource from /usr/share/variety/plugins/quotes/LocalFilesSource.py'
INFO: 2020-02-15 09:22:08,891: load() 'Jumble found plugin class: <class 'LocalFilesSource.LocalFilesSource'>: {'author': 'Peter Levi', 'name': 'Local text files', 'version': '0.1', 'description': u'Displays quotes, defined in local text files.\nPut your own txt files in: ~/.config/variety/pluginconfig/quotes/.\nThe file format is:\n\nquote -- author\n.\nsecond quote -- another author\n.\netc...\n\nExample: http://rvelthuis.de/zips/quotes.txt'}'
INFO: 2020-02-15 09:22:08,892: _walk_modules() 'Jumble loading module in __init__ from /usr/share/variety/plugins/quotes/__init__.py'
INFO: 2020-02-15 09:22:08,897: _walk_modules() 'Jumble loading module in QuotesDaddySource from /usr/share/variety/plugins/quotes/QuotesDaddySource.py'
INFO: 2020-02-15 09:22:08,926: load() 'Jumble found plugin class: <class 'QuotesDaddySource.QuotesDaddySource'>: {'author': 'Peter Levi', 'name': 'QuotesDaddy', 'version': '0.1', 'description': u"Fetches quotes from QuotesDaddy's daily quotes RSS feed.\nDoes not support searching by tags or authors."}'
INFO: 2020-02-15 09:22:08,926: _walk_modules() 'Jumble loading module in GoodreadsSource from /usr/share/variety/plugins/quotes/GoodreadsSource.py'
INFO: 2020-02-15 09:22:08,930: _walk_modules() 'Jumble loading module in QuotationsPageSource from /usr/share/variety/plugins/quotes/QuotationsPageSource.py'
INFO: 2020-02-15 09:22:09,222: reload_config() 'Creating new downloader for type 19, location Recommended by Variety. Adapts to your taste as you mark images as favorite or trash.'
ERROR: 2020-02-15 09:22:09,222: reload_config() 'Could not create Downloader for type 19, location Recommended by Variety. Adapts to your taste as you mark images as favorite or trash.'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/variety/VarietyWindow.py", line 420, in reload_config
    dlr = self.create_downloader(type, location)
  File "/usr/lib/python2.7/dist-packages/variety/VarietyWindow.py", line 555, in create_downloader
    raise Exception('No Smart user yet, not a problem')
Exception: No Smart user yet, not a problem
INFO: 2020-02-15 09:22:09,259: reload_config() 'Creating new downloader for type 6, location Random wallpapers from Desktoppr.co'
INFO: 2020-02-15 09:22:09,259: reload_config() 'Creating new downloader for type 15, location Bing Photo of the Day'
INFO: 2020-02-15 09:22:09,259: reload_config() 'Creating new downloader for type 16, location High-resolution photos from Unsplash.com'
INFO: 2020-02-15 09:22:09,259: reload_config() 'Creating new downloader for type 8, location NASA's Astronomy Picture of the Day'
INFO: 2020-02-15 09:22:09,260: reload_config() 'Creating new downloader for type 7, location user:www.flickr.com/photos/peter-levi/;user_id:93647178@N00;'
INFO: 2020-02-15 09:22:09,261: log_options() 'Loaded options:'
INFO: 2020-02-15 09:22:09,261: log_options() 'change_enabled = False'
INFO: 2020-02-15 09:22:09,261: log_options() 'change_interval = 300'
INFO: 2020-02-15 09:22:09,261: log_options() 'change_on_start = False'
INFO: 2020-02-15 09:22:09,261: log_options() 'clipboard_enabled = False'
INFO: 2020-02-15 09:22:09,261: log_options() 'clipboard_hosts = [u'wallhaven.cc', u'ns223506.ovh.net', u'wallpapers.net', u'flickr.com', u'imgur.com', u'deviantart.com', u'interfacelift.com', u'vladstudio.com', u'imageshack.us']'
INFO: 2020-02-15 09:22:09,262: log_options() 'clipboard_use_whitelist = True'
INFO: 2020-02-15 09:22:09,262: log_options() 'clock_date_font = Ubuntu Condensed, 30'
INFO: 2020-02-15 09:22:09,262: log_options() 'clock_enabled = False'
INFO: 2020-02-15 09:22:09,262: log_options() 'clock_filter = -density 100 -font `fc-match -f '%{file[0]}' '%CLOCK_FONT_NAME'` -pointsize %CLOCK_FONT_SIZE -gravity SouthEast -fill '#00000044' -annotate 0x0+[%HOFFSET+58]+[%VOFFSET+108] '%H:%M' -fill white -annotate 0x0+[%HOFFSET+60]+[%VOFFSET+110] '%H:%M' -font `fc-match -f '%{file[0]}' '%DATE_FONT_NAME'` -pointsize %DATE_FONT_SIZE -fill '#00000044' -annotate 0x0+[%HOFFSET+58]+[%VOFFSET+58] '%A, %B %d' -fill white -annotate 0x0+[%HOFFSET+60]+[%VOFFSET+60] '%A, %B %d''
INFO: 2020-02-15 09:22:09,262: log_options() 'clock_font = Ubuntu Condensed, 70'
INFO: 2020-02-15 09:22:09,262: log_options() 'configfile = /home/ej/.config/variety/variety.conf'
INFO: 2020-02-15 09:22:09,262: log_options() 'copyto_enabled = False'
INFO: 2020-02-15 09:22:09,263: log_options() 'copyto_folder = Default'
INFO: 2020-02-15 09:22:09,263: log_options() 'desired_color = [160, 160, 160]'
INFO: 2020-02-15 09:22:09,263: log_options() 'desired_color_enabled = False'
INFO: 2020-02-15 09:22:09,263: log_options() 'download_enabled = True'
INFO: 2020-02-15 09:22:09,263: log_options() 'download_folder = /home/ej/.config/variety/Downloaded'
INFO: 2020-02-15 09:22:09,263: log_options() 'download_interval = 600'
INFO: 2020-02-15 09:22:09,263: log_options() 'facebook_message = '
INFO: 2020-02-15 09:22:09,263: log_options() 'facebook_show_dialog = True'
INFO: 2020-02-15 09:22:09,264: log_options() 'favorites_folder = /home/ej/.config/variety/Favorites'
INFO: 2020-02-15 09:22:09,264: log_options() 'favorites_operations = [[u'Downloaded', u'Copy'], [u'Fetched', u'Move'], [u'Others', u'Copy']]'
INFO: 2020-02-15 09:22:09,264: log_options() 'fetched_folder = /home/ej/.config/variety/Fetched'
INFO: 2020-02-15 09:22:09,264: log_options() 'filters = [[False, u'Keep original', u''], [False, u'Grayscale', u'-type Grayscale'], [False, u'Heavy blur', u'-blur 120x40'], [False, u'Soft blur', u'-blur 20x7'], [False, u'Oil painting', u'-paint 8'], [False, u'Pencil sketch', u'-colorspace gray \\( +clone -tile ~/.config/variety/pencil_tile.png -draw "color 0,0 reset" +clone +swap -compose color_dodge -composite \\) -fx \'u*.2+v*.8\''], [False, u'Pointilism', u'-spread 10 -noise 3'], [False, u'Pixellate', u'-scale 3% -scale 3333%']]'
INFO: 2020-02-15 09:22:09,264: log_options() 'icon = Light'
INFO: 2020-02-15 09:22:09,264: log_options() 'lightness_enabled = False'
INFO: 2020-02-15 09:22:09,264: log_options() 'lightness_mode = 0'
INFO: 2020-02-15 09:22:09,265: log_options() 'min_rating = 4'
INFO: 2020-02-15 09:22:09,265: log_options() 'min_rating_enabled = False'
INFO: 2020-02-15 09:22:09,265: log_options() 'min_size = 80'
INFO: 2020-02-15 09:22:09,265: log_options() 'min_size_enabled = False'
INFO: 2020-02-15 09:22:09,265: log_options() 'quota_enabled = True'
INFO: 2020-02-15 09:22:09,265: log_options() 'quota_size = 500'
INFO: 2020-02-15 09:22:09,265: log_options() 'quotes_authors = '
INFO: 2020-02-15 09:22:09,265: log_options() 'quotes_bg_color = [80, 80, 80]'
INFO: 2020-02-15 09:22:09,265: log_options() 'quotes_bg_opacity = 55'
INFO: 2020-02-15 09:22:09,266: log_options() 'quotes_change_enabled = False'
INFO: 2020-02-15 09:22:09,266: log_options() 'quotes_change_interval = 300'
INFO: 2020-02-15 09:22:09,266: log_options() 'quotes_disabled_sources = [u'QuotesDaddy']'
INFO: 2020-02-15 09:22:09,266: log_options() 'quotes_enabled = False'
INFO: 2020-02-15 09:22:09,266: log_options() 'quotes_favorites_file = /home/ej/.config/variety/favorite_quotes.txt'
INFO: 2020-02-15 09:22:09,266: log_options() 'quotes_font = Bitstream Charter 30'
INFO: 2020-02-15 09:22:09,266: log_options() 'quotes_hpos = 100'
INFO: 2020-02-15 09:22:09,266: log_options() 'quotes_tags = '
INFO: 2020-02-15 09:22:09,267: log_options() 'quotes_text_color = [255, 255, 255]'
INFO: 2020-02-15 09:22:09,267: log_options() 'quotes_text_shadow = False'
INFO: 2020-02-15 09:22:09,267: log_options() 'quotes_vpos = 40'
INFO: 2020-02-15 09:22:09,267: log_options() 'quotes_width = 70'
INFO: 2020-02-15 09:22:09,267: log_options() 'safe_mode = False'
INFO: 2020-02-15 09:22:09,267: log_options() 'slideshow_custom_enabled = False'
INFO: 2020-02-15 09:22:09,267: log_options() 'slideshow_custom_folder = /home/ej/Pictures'
INFO: 2020-02-15 09:22:09,267: log_options() 'slideshow_downloads_enabled = False'
INFO: 2020-02-15 09:22:09,268: log_options() 'slideshow_fade = 0.4'
INFO: 2020-02-15 09:22:09,268: log_options() 'slideshow_favorites_enabled = True'
INFO: 2020-02-15 09:22:09,268: log_options() 'slideshow_mode = Fullscreen'
INFO: 2020-02-15 09:22:09,268: log_options() 'slideshow_monitor = All'
INFO: 2020-02-15 09:22:09,268: log_options() 'slideshow_pan = 0.05'
INFO: 2020-02-15 09:22:09,268: log_options() 'slideshow_seconds = 6.0'
INFO: 2020-02-15 09:22:09,268: log_options() 'slideshow_sort_order = Random'
INFO: 2020-02-15 09:22:09,268: log_options() 'slideshow_sources_enabled = True'
INFO: 2020-02-15 09:22:09,269: log_options() 'slideshow_zoom = 0.2'
INFO: 2020-02-15 09:22:09,269: log_options() 'smart_enabled = True'
INFO: 2020-02-15 09:22:09,269: log_options() 'smart_notice_shown = True'
INFO: 2020-02-15 09:22:09,269: log_options() 'smart_register_shown = True'
INFO: 2020-02-15 09:22:09,269: log_options() 'sources = [[True, 3, u'The Favorites folder'], [True, 4, u'The Fetched folder'], [True, 19, u'Recommended by Variety. Adapts to your taste as you mark images as favorite or trash.'], [False, 20, u'Latest favorites by the other users of Variety. [May contain NSFW images]'], [True, 2, u'/usr/share/backgrounds'], [True, 6, u'Random wallpapers from Desktoppr.co'], [True, 15, u'Bing Photo of the Day'], [True, 16, u'High-resolution photos from Unsplash.com'], [True, 8, u"NASA's Astronomy Picture of the Day"], [False, 11, u'World Sunlight Map - live wallpaper from Die.net'], [True, 7, u'user:www.flickr.com/photos/peter-levi/;user_id:93647178@N00;']]'
INFO: 2020-02-15 09:22:09,269: log_options() 'stats_enabled = True'
INFO: 2020-02-15 09:22:09,269: log_options() 'stats_notice_shown = True'
INFO: 2020-02-15 09:22:09,269: log_options() 'sync_enabled = True'
INFO: 2020-02-15 09:22:09,270: log_options() 'use_landscape_enabled = True'
INFO: 2020-02-15 09:22:09,270: reload_config() 'No need to clear prepared queue'
INFO: 2020-02-15 09:22:09,271: load_last_change_time() 'Change interval < 6 hours, ignore persisted last_change_time, wait initially the whole interval: 300'
INFO: 2020-02-15 09:22:09,271: update_indicator() 'Setting file info to: /usr/share/backgrounds/linuxmint-tara/jowens_kauai.jpg'
INFO: 2020-02-15 09:22:09,372: do_set_wp() 'Calling do_set_wp with /usr/share/backgrounds/linuxmint-tara/jowens_kauai.jpg, time: 1581776529.37'
INFO: 2020-02-15 09:22:09,404: regular_change_thread() 'regular_change thread running'
INFO: 2020-02-15 09:22:09,405: prepare_thread() 'Prepare thread running'
INFO: 2020-02-15 09:22:09,405: regular_change_thread() 'regular_change: waiting till user resumes'
INFO: 2020-02-15 09:22:09,406: prepare_thread() 'Prepared buffer contains 0 images'
INFO: 2020-02-15 09:22:09,408: prepare_thread() 'Preparing some images'
INFO: 2020-02-15 09:22:09,433: _run() 'sync: Started, hash 7967dc6657bbbe16c6fa92365905351c'
INFO: 2020-02-15 09:22:09,434: load_user() 'smart: Missing smart_user.json, creating new smart user'
INFO: 2020-02-15 09:22:09,435: new_user() 'smart: Creating new smart user'
INFO: 2020-02-15 09:22:09,435: update_indicator_icon() 'Creating indicator'
INFO: 2020-02-15 09:22:09,565: update_indicator() 'Setting file info to: /usr/share/backgrounds/linuxmint-tara/jowens_kauai.jpg'
WARNING: 2020-02-15 09:22:09,579: create_menu() 'Variety Slideshow is not installed. This is an optional extension adding pan-and-zoom slideshows to Variety: see https://github.com/peterlevi/variety-slideshow for details'
INFO: 2020-02-15 09:22:09,966: set_from_theme_icon() 'Showing indicator icon variety-indicator from GTK theme'
INFO: 2020-02-15 09:22:09,966: update_indicator() 'Setting file info to: /usr/share/backgrounds/linuxmint-tara/jowens_kauai.jpg'
ERROR: 2020-02-15 09:22:11,233: set_desktop_wallpaper() 'Exception when calling set_wallpaper script: 127'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/variety/VarietyWindow.py", line 2532, in set_desktop_wallpaper
    subprocess.check_call(["timeout", "--kill-after=5", "10", script, wallpaper, auto, original_file])
  File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
    raise CalledProcessError(retcode, cmd)
CalledProcessError: Command '['timeout', '--kill-after=5', '10', u'/home/ej/.config/variety/scripts/set_wallpaper', u'/usr/share/backgrounds/linuxmint-tara/jowens_kauai.jpg', 'refresh', u'/usr/share/backgrounds/linuxmint-tara/jowens_kauai.jpg']' returned non-zero exit status 127
INFO: 2020-02-15 09:22:11,959: reload() 'Reloading preferences dialog'
INFO: 2020-02-15 09:22:12,731: prepare_thread() 'After search prepared buffer contains 99 images'
INFO: 2020-02-15 09:22:19,569: load_user() 'smart: Missing smart_user.json, creating new smart user'
INFO: 2020-02-15 09:22:19,569: new_user() 'smart: Creating new smart user'
ERROR: 2020-02-15 09:22:19,568: _run() 'sync: Could not load or create smart user'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/variety/Smart.py", line 516, in _run
    self.load_user(create_if_missing=True)
  File "/usr/lib/python2.7/dist-packages/variety/Smart.py", line 128, in load_user
    self.new_user()
  File "/usr/lib/python2.7/dist-packages/variety/Smart.py", line 136, in new_user
    self.user = Util.fetch_json(Smart.API_URL + '/newuser')
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 529, in fetch_json
    return Util.request(url, data).json()
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 506, in request
    verify=False)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 520, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 630, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/adapters.py", line 508, in send
    raise ConnectionError(e, request=request)
ConnectionError: HTTPSConnectionPool(host='vrty.org', port=443): Max retries exceeded with url: /api/newuser (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3dadf4c2d0>: Failed to establish a new connection: [Errno -2] Name or service not known',))
INFO: 2020-02-15 09:22:20,474: load_user() 'smart: Missing smart_user.json, creating new smart user'
INFO: 2020-02-15 09:22:20,475: new_user() 'smart: Creating new smart user'
INFO: 2020-02-15 09:22:20,480: load_user() 'smart: Missing smart_user.json, creating new smart user'
INFO: 2020-02-15 09:22:20,481: new_user() 'smart: Creating new smart user'
INFO: 2020-02-15 09:22:20,486: _run() 'sync: Started, hash 5e0cdf9b449aecf16dabe0209ccb3945'
INFO: 2020-02-15 09:22:20,487: load_user() 'smart: Missing smart_user.json, creating new smart user'
INFO: 2020-02-15 09:22:20,487: new_user() 'smart: Creating new smart user'
ERROR: 2020-02-15 09:22:20,494: _run() 'sync: Could not load or create smart user'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/variety/Smart.py", line 516, in _run
    self.load_user(create_if_missing=True)
  File "/usr/lib/python2.7/dist-packages/variety/Smart.py", line 128, in load_user
    self.new_user()
  File "/usr/lib/python2.7/dist-packages/variety/Smart.py", line 136, in new_user
    self.user = Util.fetch_json(Smart.API_URL + '/newuser')
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 529, in fetch_json
    return Util.request(url, data).json()
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 506, in request
    verify=False)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 520, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 630, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/adapters.py", line 508, in send
    raise ConnectionError(e, request=request)
ConnectionError: HTTPSConnectionPool(host='vrty.org', port=443): Max retries exceeded with url: /api/newuser (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3dac5dbdd0>: Failed to establish a new connection: [Errno -2] Name or service not known',))
INFO: 2020-02-15 09:22:20,523: update_indicator() 'Setting file info to: /usr/share/backgrounds/linuxmint-tara/jowens_kauai.jpg'
INFO: 2020-02-15 09:22:29,288: stats_report_config() 'Stats: Reporting config anonymously'
ERROR: 2020-02-15 09:22:29,386: reporting_thread_method() 'Stats: Could not report config'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/variety/VarietyWindow.py", line 999, in reporting_thread_method
    self.smart.stats_report_config()
  File "/usr/lib/python2.7/dist-packages/variety/Smart.py", line 851, in stats_report_config
    res = Util.fetch_json(Smart.API_URL + '/stats/%s/report-config' % statsid, data=data)
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 529, in fetch_json
    return Util.request(url, data).json()
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 506, in request
    verify=False)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 520, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 630, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/adapters.py", line 508, in send
    raise ConnectionError(e, request=request)
ConnectionError: HTTPSConnectionPool(host='vrty.org', port=443): Max retries exceeded with url: /api/stats/40h04ukfx1/report-config (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3d79c13250>: Failed to establish a new connection: [Errno -2] Name or service not known',))
INFO: 2020-02-15 09:22:29,432: server_options_thread() 'Fetching server options from http://tiny.cc/variety-options-063'
INFO: 2020-02-15 09:22:29,978: server_options_thread() 'Fetched server options: {u'unsplash': {u'min_fill_queue_interval': 20000000000, u'min_download_interval': 20000000000}, u'unsplash_v2': {u'min_fill_queue_interval': 1800, u'min_download_interval': 600}, u'wallpapers.net': {u'min_fill_queue_interval': 600, u'min_download_interval': 0}, u'wallhaven': {u'min_fill_queue_interval': 1800, u'min_download_interval': 1800}, u'wallbase': {u'min_fill_queue_interval': 1800, u'min_download_interval': 1800}, u'flickr': {u'min_fill_queue_interval': 1200, u'min_download_interval': 1200}, u'status_message': {u'*': u'', u'0.0.1': u''}}'
INFO: 2020-02-15 09:22:59,698: stats_report_config() 'Stats: Reporting config anonymously'
ERROR: 2020-02-15 09:23:09,708: reporting_thread_method() 'Stats: Could not report config'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/variety/VarietyWindow.py", line 999, in reporting_thread_method
    self.smart.stats_report_config()
  File "/usr/lib/python2.7/dist-packages/variety/Smart.py", line 851, in stats_report_config
    res = Util.fetch_json(Smart.API_URL + '/stats/%s/report-config' % statsid, data=data)
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 529, in fetch_json
    return Util.request(url, data).json()
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 506, in request
    verify=False)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 520, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 630, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/adapters.py", line 508, in send
    raise ConnectionError(e, request=request)
ConnectionError: HTTPSConnectionPool(host='vrty.org', port=443): Max retries exceeded with url: /api/stats/40h04ukfx1/report-config (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3dadf4ce50>: Failed to establish a new connection: [Errno -2] Name or service not known',))
INFO: 2020-02-15 09:23:39,735: stats_report_config() 'Stats: Reporting config anonymously'
ERROR: 2020-02-15 09:23:54,747: reporting_thread_method() 'Stats: Could not report config'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/variety/VarietyWindow.py", line 999, in reporting_thread_method
    self.smart.stats_report_config()
  File "/usr/lib/python2.7/dist-packages/variety/Smart.py", line 851, in stats_report_config
    res = Util.fetch_json(Smart.API_URL + '/stats/%s/report-config' % statsid, data=data)
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 529, in fetch_json
    return Util.request(url, data).json()
  File "/usr/lib/python2.7/dist-packages/variety/Util.py", line 506, in request
    verify=False)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 520, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 630, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/adapters.py", line 508, in send
    raise ConnectionError(e, request=request)
ConnectionError: HTTPSConnectionPool(host='vrty.org', port=443): Max retries exceeded with url: /api/stats/40h04ukfx1/report-config (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3d79c13e50>: Failed to establish a new connection: [Errno -2] Name or service not known',))
INFO: 2020-02-15 09:32:09,527: purge_downloaded() 'Refreshed download folder size: 18 mb'
INFO: 2020-02-15 09:32:09,527: download_one() 'Downloading an image from NASA's Astro Pic of the Day, nasa_apod'
INFO: 2020-02-15 09:32:09,527: download_one() 'Queue size: 0'
INFO: 2020-02-15 09:32:09,528: fill_queue() 'Filling APOD queue from Archive'
INFO: 2020-02-15 09:32:10,393: fill_queue() 'APOD queue populated with 13 URLs'
INFO: 2020-02-15 09:32:10,393: download_one() 'APOD URL: http://apod.nasa.gov/apod/ap200215.html'
INFO: 2020-02-15 09:32:10,581: download_one() 'Image URL: http://apod.nasa.gov/apod/image/2002/Eta-HST-ESO-New-LL.jpg'
INFO: 2020-02-15 09:32:10,581: save_locally() 'Origin URL: http://apod.nasa.gov/apod/ap200215.html'
INFO: 2020-02-15 09:32:10,582: save_locally() 'Image URL: http://apod.nasa.gov/apod/image/2002/Eta-HST-ESO-New-LL.jpg'
INFO: 2020-02-15 09:32:10,582: save_locally() 'Local name: /home/ej/.config/variety/Downloaded/nasa_apod/Eta-HST-ESO-New-LL.jpg'
INFO: 2020-02-15 09:32:11,808: save_locally() 'Download complete'
INFO: 2020-02-15 09:32:11,809: update_indicator() 'Setting file info to: /usr/share/backgrounds/linuxmint-tara/jowens_kauai.jpg'
INFO: 2020-02-15 09:32:11,810: download_one_from() 'Adding downloaded file /home/ej/.config/variety/Downloaded/nasa_apod/Eta-HST-ESO-New-LL.jpg to prepared_from_downloads queue'
